nx-X11/programs/Xserver/hw/nxagent/Pixmap.c

```diff
@@ -116,7 +111,7 @@ PixmapPtr nxagentCreatePixmap(ScreenPtr pScreen, int width, int height,
    * not allocate memory for the data.
    */
 
-  pPixmap = AllocatePixmap(pScreen, 0);
+  PixmapPtr pPixmap = AllocatePixmap(pScreen, 0);
 
```

Comment:
Don't know if this is a good idea. I have seen compiler warnings with GCC-9 that complain about inline variable declarations. I'd leave them at the top of the scope (i.e. the function).